service/discovery/discovery.go

```diff
@@ -123,19 +123,19 @@
 	// create an authorizer from env vars or Azure Managed Service Identity
 	authorizer, err := auth.NewAuthorizerFromCLI()
 	if err != nil {
-		log.Errorf("Could not authenticate to Azure: %s", err)
+		log.Errorf("Could not authenticate to Azure: %v", err)
 		return nil, err
```

Comment:
Why not wrap error with `%w` like in k8s compute? There, we don't print the error which is thrown to the calling function. Maybe we have to discuss this and agree on a common way again next year and then do a project-wide fix.

Reply:
You are right, the `log.Errorf`must all be `%w` instead of `%v`.

Reply:
Ah, I think the the confusion regarding service/k8s/compute.go was `log.Errorf` vs. `fmt.Errorf`?